daemon/daemon_test.go

```diff
@@ -0,0 +1,13 @@
+package main_test
```

Comment:
Why do we need a main test? Why can't this test be called vendorplugin_test?

Reply:
We're actually testing the daemon, and since daemon is in main, I've used main_test.